metricbeat/module/docker/diskio/data.go

```diff
@@ -0,0 +1,30 @@
+package diskio
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+)
+
+func eventsMapping(blkioStatsList []BlkioStats) []common.MapStr {
+	myEvents := []common.MapStr{}
+	for _, blkioStats := range blkioStatsList {
+		myEvents = append(myEvents, eventMapping(&blkioStats))
+	}
+	return myEvents
+}
+func eventMapping(myBlkioStats *BlkioStats) common.MapStr {
+	event := common.MapStr{
+		"@timestamp": myBlkioStats.Time,
```

Comment:
@ruflin , timestamp is not now(). It's the time generate by Docker daemon while sending the events requested! we assume that it's better to use this time because it's ++exact!